readme.md

```diff
@@ -1,17 +1,21 @@
-# Service Fabric Cluster with Datadog installed as a VM extension on each node
+# Datadog Service Fabric Cluster Template
 
-These ARM templates set up unsecure Service Fabric clusters that include the Datadog agent on each node as a VM extension.
+These ARM templates set up an insecure Service Fabric cluster that includes the Datadog Agent on each node as a VM extension.
 
-## PowerShell commands to run:
+## Usage
 
-```powershell
-Login-AzureRmAccount
+There are two templates, for both Windows and Linux.
```

Comment:
```suggestion
There are two templates each for both Windows and Linux.
```

Reply:
The intended meaning was to say that there is one template for Windows VMs and one for Linux VMs, I reworded the sentence so that it's a bit more clear.